java-squid/src/main/java/org/sonar/plugins/java/api/JavaFileScannerContext.java

```diff
@@ -37,6 +37,7 @@
 
   void addIssue(Tree tree, JavaCheck check, String message);
 
+  @Deprecated
   void addIssue(Tree tree, JavaCheck check, String message, @Nullable Double cost);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)